pcsx2-qt/Translations.cpp

```diff
@@ -121,7 +121,11 @@ void QtHost::InstallTranslator(QWidget* dialog_parent)
 #ifdef __APPLE__
 	const QString base_dir = QStringLiteral("%1/../Resources/translations").arg(qApp->applicationDirPath());
 #else
-	const QString base_dir = QStringLiteral("%1/translations").arg(qApp->applicationDirPath());
+	#ifdef PCSX2_APP_DATADIR
```

Comment:
Since there's already an ifdef above this, could you change it to `#if defined(__APPLE__) ... #elif defined(PCSX2_APP_DATADIR) ... #else`?

Reply:
Done